Fix malformed technique links

EricDunsworth opened the pull request.

alastc commented:
Question for @michael-n-cooper - these are empty & broken links (from checking a couple, e.g. [SCR1](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR1)), is it a build issue?

E.g. `(see <a href="#SCR16"></a>)`

Is the build script supposed to do something with those?

awkawk commented:
The links don't seem broken to me: https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR1 - the link to SCR16 in the description section goes to SCR16.

awkawk commented:
oops - wrong section. I see the issue.

EricDunsworth commented:
FWIW this PR covers 2 "kinds" of link patterns...

**G209 and G214's changes:**
  * Links were initially coded like this:
``<a href="G207.html">G207</a>``
  * Which are generated like this:
``<a href="https://w3c.github.io/wcag/techniques//G207.html">G207.html: </a>``
  * Example:
[G209's related techniques section](https://w3c.github.io/wcag/techniques/general/G209.html#related)

**All other changes:**
* Links were initially coded like this:
``<a href="#H71"></a>``
* Which are generated like this (unchanged):
``<a href="#H71"></a>``
* Example:
[H44's third example (check the note box)](https://w3c.github.io/wcag/techniques/html/H44.html#example-3-a-group-of-radio-buttons)

EricDunsworth commented:
Just fixed the merge conflict (removed a Flash technique file) and fixed the broken skip links in F72 / H86.

Also, I just wanted to note that all of the links this PR fixes are still broken to this day.

The broken links are also present in [the W3C website's versions of the WCAG 2.1+2.2 techniques](https://www.w3.org/WAI/WCAG22/Techniques/).
